File: net/Postpony.Api.Sdk/Postpony.Api.Sdk/Model/ShippingDetailResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace Postpony.Api.Sdk.Model
{
    [XmlRoot("ShippingDetailResponseV2")]
    public class ShippingDetailResponse:ResponseBase
    {

        public LabelInfoDetail Detail { set; get; }

    }
}

File: java/postpony-api-sdk/src/main/java/com/postpony/model/ShippingDetailResponse.java
package com.postpony.model;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement(name="ShippingDetailResponseV2")
public class ShippingDetailResponse extends ResponseBase{

    public LabelInfoDetail getDetail() {
        return detail;
    }

    public void setDetail(LabelInfoDetail detail) {
        this.detail = detail;
    }

    @XmlElement(name = "Detail")
    LabelInfoDetail detail;
}
